haystack/utils/deepsetcloud.py

```diff
@@ -865,9 +865,50 @@ def _get_evaluation_sets(self, workspace: Optional[str] = None) -> Generator:
         evaluation_set_url = f"{url}/evaluation_sets"
         return self.client.get_with_auto_paging(url=evaluation_set_url)
 
-    def _get_evaluation_set(
+    def upload_evaluation_set(self, file_path: Path, workspace: Optional[str] = None):
+        """
+        Uploads an evaluation set file.
+        The evaluation set will be accessible under the name of the uploaded file.
```

Comment:
The name of file that you uploaded becomes the name of the evaluation set in deepset Cloud.